rentals.json

```diff
@@ -0,0 +1 @@
+[{"id":1,"book_id":101,"person_id":201,"due_date":"2023-12-31","return_date":null},{"id":2,"book_id":102,"person_id":202,"due_date":"2023-11-15","return_date":"2023-11-10"}]
```

Comment:
- [ ] I couldn't test adding and displaying rentals due to the other errors. Kindly make sure it works as intended.